packages/devtools_app/lib/src/bloc/bloc/bloc_list_bloc.dart

```diff
@@ -0,0 +1,82 @@
+import 'dart:async';
+
+import 'package:bloc/bloc.dart';
+import 'package:vm_service/vm_service.dart';
+
+import '../../eval_on_dart_library.dart';
+import '../../globals.dart';
+import '../models/bloc_object.dart';
+
+part 'bloc_list_event.dart';
+part 'bloc_list_state.dart';
+
+class BlocListBloc extends Bloc<BlocListEvent, BlocListState> {
+  BlocListBloc() : super(const BlocListInitial()) {
```

Comment:
the service manager service instance should probably be injectable so that it can be mocked for testing purposes.